src/App.js

```diff
@@ -1,8 +1,14 @@
+/** @format */
+
+import { Provider } from 'react-redux';
 import './App.scss';
 import Navbar from './components/Navbar';
+import store from './redux/configureStore';
 
-function App() {
-  return <Navbar />;
-}
+const App = () => (
+  <Provider store={store}>
+    <Navbar />
+  </Provider>
+);
```

Comment:
- [ ] The `<Navbar />` component should be responsible only for navigation, kindly create a new component separating the `<Navbar />` and the main home page of your project then update your components routing.
Should be something like this:
```jsx
<Provider store={store}>
  <Navbar />
  <Router>
    <Switch>
      <Route path="/books">
        <Books />
      </Route>
      <Route path="/categories">
        <Categories />
      </Route>
    </Switch>
  </Router>
</Provider>
```
- [ ] Please update your routes to route the visitor directly to the books page as the `path='/'`, currently, I have to click on the `Books` link to get there.